### ISSUE TITLE
problem about the reslut

### ISSUE THREAD

**Issue by zyfsa** [JiaRenChang/PSMNet#27]
hello, 
1. do you test the result of model only trained the scene flow on the kitti2015-all-traing-image? the result is?
2.when you finetune on the kitti2015-traing, the result of model on the kitti2015-all-traing-image is?

my problom is when I finetune on the kitti-2015-traing, then test the result on the kitti2015-all-traing-image, the relslut has little change after 30epoch， now the result of err_rate is about 3.4%? do you have other special trainging method? I am chinese, we can chat in weixin if you want, thank you!
Please! I am troubled in there, the performance is pool.

**Comment by JiaRenChang** [comment]
@zyfsa 
Hi, 請問你是下載了從SceneFlow後從頭開始練嗎?

**Comment by zyfsa** [comment]
hi, @JiaRenChang  我们是简体，你是繁体，希望能看懂，哈哈。我就是下载了sceneflow重头开始训练，得到了预训练的模型；然后在finetune,我发现我finuetune有一定的效果，但是在30个epoch后就不变了，但是在kitti2015训练集上错误率3.5左右，我所以想知道一下您的中间结果，进行比较。
另外您在sceneflow训练的时候是剔除了>192的点，在kitti时剔除了<0的点，这一步是很重要吧？ 我也是这样做的。能帮我解答一下我的结果为什么是这样吗？

**Comment by JiaRenChang** [comment]
@zyfsa 
那請問您訓練出的模型在SceneFlow的測試集上的結果如何？ 是否符合論文中的結果？

**Comment by zyfsa** [comment]
是不太符合的，EPE没有你那么低；我用这个预训练的模型直接在kitti2015上跑，错误率20%;然后在整个kitti上finetune,错误率是3.5%.我不知道你是否有像我这么测试过，结果如何；另外你最后得到的视差图是1/4分辨率训练的loss，然后直接上采样到原始图，精度都有那么高，确实厉害。另外麻烦回答一下，剔除某些点得到loss训练是否很重要

**Comment by zyfsa** [comment]
我没有做augmentation,只是把数据预处理到-1到1的值范围

**Comment by zyfsa** [comment]
另外我想请问你训练的时候除了smooth l1_loss,有weight decay 吗？是否有影响

**Comment by JiaRenChang** [comment]
Hi,
我們前處理有做color normalization，用的mean/variance是來自於imagenet
訓練時有做random crop 隨機裁切成(256, 512)的影像做訓練
沒有做weight decay

**Comment by JiaRenChang** [comment]
We usually scale disparities because we scaled image sizes.
For example, if we re-scale image to half, the disparity value should be half because that the correspondences are closer.

**Comment by JiamingSuen** [comment]
Yes, I get that, while it's probably not the answer to my question.
Thanks for your reply anyway.

**Comment by DengYon** [comment]
@JiamingSuen Do you have answer to your question now?

**Comment by JiamingSuen** [comment]
I'm afraid not, it seems that few people have tried that. According to my experiment it will not improve the result a lot, but this is far from a solid conclusion.
